src/main/java/uk/gov/justice/laa/amend/claim/utils/DateUtils.java

```diff
@@ -36,4 +39,12 @@ public static String toSubmissionPeriod(String month, String year) {
             return null;
         }
     }
+
+    public static String getDateValue(OffsetDateTime localDate) {
+        return localDate != null ? localDate.format(DateTimeFormatter.ofPattern(DEFAULT_DATE_FORMAT)) : null;
+    }
+
+    public static String getTimeValue(OffsetDateTime localDate) {
+        return localDate != null ? localDate.format(DateTimeFormatter.ofPattern(DEFAULT_TIME_FORMAT)) : null;
+    }
```

Comment:
Could possibly look to refactor this so we don't need to split out the date and time. See `displayDateTimeValue` for an example (could convert the `lastAssessedDate` to a `LocalDateTime` in the mapper, similar to what we do with the `submittedDate`)